### PR TITLE
DAT-2627 remove title from hero module

### PR DESCRIPTION
**Pull request by RafLeszczynski**
Please do not merge - waiting for mocks from design to modify UI

### CODE REVIEW

File: extensions/wikia/NjordPrototype/NjordController.class.php [hunk 1]
```diff
@@ -167,21 +167,17 @@ public function upload() {
 	}
 
 	public function saveTitleAndDescription() {
-		$title = $this->getRequest()->getVal( 'title', false );
+		//$title = $this->getRequest()->getVal( 'title', false );
```
Comment (SebastianMarzjan) [inline extensions/wikia/NjordPrototype/NjordController.class.php]: I think you can just remove this line

**Comment by SebastianMarzjan** [comment]
Seems OK for the current stage (complete rewrite upcoming), though please have a second pair of eyes go through this.

**Comment by RafLeszczynski** [comment]
@SebastianMarzjan applied UI changes requested by design, please take a look ones again.

**Comment by SebastianMarzjan** [comment]
let it be

**Comment by SebastianMarzjan** [comment]
Purely styling changes, gtqa

**Pull request merged**